### PR TITLE
Remove inline js

### PR DESCRIPTION
**Pull request by nikkolai14**
#13270

### CODE REVIEW

**Comment by williamdes** [comment]
Hi @nikkolai14  
Thank you for contributing!
I would recommend you to use some git visualisation tool like GitKraken in order to avoid doing merges of merges but instead one commit or more on your pull-requests :)

**Comment by nikkolai14** [comment]
@williamdes Okay, I will try to learn the GitKraken tool, sorry about that.

File: templates/export.twig [hunk 1]
```diff
@@ -421,6 +421,6 @@
   <div class="exportoptions justify-content-end" id="submit">
     <input id="buttonGo" class="btn btn-primary" type="submit" value="{% trans 'Go' %}"
       {#- If the time limit set is zero, then time out won't occur so no need to check for time out. -#}
-      {%- if exec_time_limit > 0 %} onclick="Export.checkTimeOut({{ exec_time_limit }})"{% endif %}>
+      {%- if exec_time_limit > 0 %} data-exec-time-limit="{{ exec_time_limit }}" {% endif %}>
```
Comment (mauriciofauth) [inline templates/export.twig:424]: I think you could move that if to the js file as well.

File: js/export.js [hunk 2]
```diff
@@ -1042,4 +1042,9 @@ AJAX.registerOnload('export.js', function () {
         e.preventDefault();
         setSelectOptions(false);
     });
+
+    $('#buttonGo').on('click', function () {
+        var timeLimit = parseInt($(this).attr('data-exec-time-limit'));
+        Export.checkTimeOut(timeLimit);
```
Comment (mauriciofauth) [inline js/export.js:1048]:
```suggestion
        if (timeLimit > 0) {
            Export.checkTimeOut(timeLimit);
        }
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by mauriciofauth** [comment]
> @williamdes Okay, I will try to learn the GitKraken tool, sorry about that.

You don't have to sorry about that. It is a recommendation to help you improve as a developer. Having a lot of commits in the pull request doesn't matter to us that much, since we can squash them before merging.

**Comment by williamdes** [comment]
> > @williamdes Okay, I will try to learn the GitKraken tool, sorry about that.
> 
> You don't have to sorry about that. It is a recommendation to help you improve as a developer. Having a lot of commits in the pull request doesn't matter to us that much, since we can squash them before merging.

I agree but I would add that it adds some overhead for me to do squashes before a merge :)
Do as you want with a good balance between not wasting your time and not adding to much overhead to the developer merging your work. Examples: good pull-request descriptions, fixes number, clean git history, etc..

Anyway, enjoy your coding time and thank you for your contributions! ;)

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**